### PR TITLE
feat(tup-cms): svg decals for layered images

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview & Changes

Allow designers to add TACC-specific decals to images.

- [Django Filer (Django CMS uses this for file management) does not support SVG.](https://github.com/django-cms/django-filer/issues/460)

## Related

- requires https://github.com/TACC/Core-CMS/pull/611

## Changes

- added SVG snippet templates

## Testing & UI

https://user-images.githubusercontent.com/62723358/226137930-5e0c2d2b-282e-4caa-b9ad-efc9e29156fe.mov

## Notes

I accidentally merged `dev/tup-cms` into here. I fixed it by merging #167 into `main`, then `main` into `dev/tup-cms` and `dev/tup-cms` into here (again). VCS is fun.

### Known Issues

1. https://github.com/django-cms/djangocms-icon/issues/9

<details><summary>Video</summary>

https://user-images.githubusercontent.com/62723358/226656807-7683437a-09bf-4e1c-9bbb-784bd4790969.mov

</details>

### CODE REVIEW

**Review by taoteg — APPROVED** [review]
Looks good. 
Do we care about the unused assignment annotation warnings in Sidebar.tsk, ProjectDetails.tsx  and Dashboard.tsx? 
They appear to be unchanged, but flagged files in the PR.
Approving regardless.

**Comment by wesleyboar** [comment]
@taoteg Annotation are unrelated. We care, but I have active work on tup-ui. When someone get's a cycle, someone will commit or open a PR will fix those.

**Comment by wesleyboar** [comment]
@taoteg Oh, never mind. There's some odd stuff in this diff. I must have accidentally merged `dev/tup-cms` into here. Yay, new task.

**Comment by wesleyboar** [comment]
Fixed. And I added some more symbols.

File: apps/tup-cms/src/taccsite_cms/settings_custom.py [hunk 1]
```diff
@@ -166,3 +172,20 @@
...
+
+from taccsite_cms.settings import DJANGOCMS_ICON_SETS as CORECMS_ICON_SETS
+
+DECAL_ICONFILE = os.path.join(BASE_DIR, 'taccsite_custom', 'tup-cms', 'static', 'tup-cms', 'img', 'icons', 'decals.json')
+with open(DECAL_ICONFILE, 'r') as fh:
+    DECAL_ICONS = fh.read()
+
+# HELP: Any CORECMS_ICON_SETS placement has a bug:
+#       - If last, editor assumes SVG is from tup-cms DECAL_ICONS.
+#       - If first, editor assumes SVG is from Core-CMS LOGO_ICONS.
```
Comment (wesleyboar) [inline apps/tup-cms/src/taccsite_cms/settings_custom.py:188]:
```suggestion
# HELP: Icon dropdown sometimes does not show active icon selected.
#       The position of DECAL_ICONS in this array affects the bug.
```

File: apps/tup-cms/src/taccsite_cms/settings_custom.py [hunk 2]
```diff
@@ -166,3 +172,19 @@
...
+# https://github.com/django-cms/djangocms-icon
+
+from taccsite_cms.settings import DJANGOCMS_ICON_SETS as CORECMS_ICON_SETS
+
+DECAL_ICONFILE = os.path.join(BASE_DIR, 'taccsite_custom', 'tup-cms', 'static', 'tup-cms', 'img', 'icons', 'decals.json')
+with open(DECAL_ICONFILE, 'r') as fh:
+    DECAL_ICONS = fh.read()
+
+# HELP: Icon dropdown sometimes does not show active icon selected.
+#       The position of DECAL_ICONS in this array affects the bug.
```
Comment (wesleyboar) [inline apps/tup-cms/src/taccsite_cms/settings_custom.py:187]:
```suggestion
# HELP: Icon dropdown sometimes does not show active icon selected.
#       The position of DECAL_ICONS in this array affects the bug.
# SEE: https://github.com/django-cms/djangocms-icon/issues/9
```

**Pull request merged**